homeassistant/components/samsungtv/media_player.py

```diff
@@ -117,17 +122,50 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
         LOGGER.info("Ignoring duplicate Samsung TV %s", host)
 
 
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Set up the Samsung TV from a config entry."""
+    host = config_entry.data.get(CONF_HOST)
```

Comment:
Please use `dict[key` for keys that we know exist in the dict.
```suggestion
    host = config_entry.data[CONF_HOST]
```